Scripts/Terminology-Install-Utilities.psm1

```
@@ -551,61 +643,68 @@ function Invoke-PostToDps {
+        $body = "{ `"DataMartId`": $dataMartId, `"BatchExecution`": { `"PipelineType`": `"Migration`", `"OverrideLoadType`": `"Incremental`", `"LoggingLevel`": `"Diagnostic`" } }"
+        Write-DosMessage -Level "Information" -Message "Scheduling job for '$name' with the Data Processing Service to creating physical tables.`n$($Config.dpsServiceUrl)/ExecuteDataMart`n$body"
+        $response = Invoke-RestMethod -Uri "$($Config.dpsServiceUrl)/ExecuteDataMart" -Method POST -UseDefaultCredentials -Headers $headers -Body $body -UseBasicParsing
         $batchExecutionId = ($response.value | ConvertFrom-Json).Id
-        Write-DosMessage -Level "Info" -Message  "Batch execution successfully sent to the data processing service."
+        Write-DosMessage -Level "Information" -Message  "Batch execution successfully sent to the data processing service."
 
         return $batchExecutionId;
     }
     catch [System.Net.WebException] {
-        Write-DosMessage -Level "Error" -Message  "POST for '$name' failed with status code:" $_.Exception.Response.StatusCode.value__ -ErrorAction Continue
-        Write-DosMessage -Level "Error" -Message  "Description:" $_.Exception.Response.StatusDescription -ErrorAction Continue
-        throw $_.Exception
+        Write-Host $_.ErrorDetails.Message | ConvertFrom-Json | Select-Object -Expand message
+        Write-Host "Second"
+        Write-Host $_.Exception.Message
+        Write-Host $_.Exception.Response
+        Write-Host $_.Exception.InnerException
+        Write-Host $_.Exception.InnerException | ConvertFrom-Json | Select-Object -Expand innerException
+        Write-Host $_.Exception | ConvertFrom-Json
+        Write-Host $_.Exception | ConvertFrom-Json | Select-Object -Expand message
+        Write-Host ($_.Exception | Format-Table | Out-String)
+        
+        #Write-DosMessage -Level "Error" -Message  "POST for '$name' failed with status code:" $_.Exception.Response.StatusCode.value__ 
```

Commented code?  Also, I believe we want to rethrow the exception.